neuralmonkey/encoders/imagenet_encoder.py

```diff
@@ -74,17 +109,23 @@ def __init__(self,
             _, end_points = net_function(self.input_plc)
 
         with tf.variable_scope(self.name):
-            net_output = end_points[output_layer]
-            if not fine_tune:
-                net_output = tf.stop_gradient(net_output)
-            # pylint: disable=no-member
-            shape = [s.value for s in net_output.get_shape()[1:]]
-            # pylint: enable=no-member
-            self.__attention_tensor = tf.reshape(
-                net_output, [-1, shape[0] * shape[1], shape[2]])
-
-            self.encoded = tf.reduce_mean(net_output, [1, 2])
-    # pylint: enable=too-many-arguments
+            if attention_layer is not None:
+                net_output = end_points[attention_layer]
```

Comment:
tady mi neni moc jasný, kde se ten `attention_layer` klíč vezme v tom `end_points`. to je nějaký standardizovaný označení? A slouží to teda k tomu, že attention se dělá nad vrstvou `attention_layer`?